src/domain/community/contributor/CommunityContributorsBlockWide/CommunityVirtualContributorsBlockWide.tsx

```diff
@@ -73,7 +66,7 @@ const CommunityVirtualContributorsBlockWide = ({
                       avatar={vc.profile.avatar?.uri ?? ''}
                       displayName={vc.profile.displayName}
                       tooltip={{ tags: [] }}
-                      url={vc.profile.url.substring(origin.length, vc.profile.url.length) ?? ''}
+                      url={buildVirtualContributorUrl(vc.nameID)}
```

Comment:
Please, request the profile { url } instead of nameID. No need of `buildVirtualContributorUrl` just pass the vc.profile.url.